src/main/java/Objects/Packets/Crypto.java

```diff
@@ -2,9 +2,11 @@
 
 public class Crypto extends Packet {
 
-    public Crypto() {
-        /*setCost(13); //TODO: Update with balanced value?
-        setStealth(0);*/
+    public Crypto(final String team) {
+        setPacketType("Crypto");
+        setTeam(team);
+        setCost(13);
+        setStealth(0);
     }
 
```

Comment:
this one too!

Reply:
With ten of these, you could fit your thumb in there.